history/cultures/eredar.txt

```diff
@@ -15,6 +15,10 @@
 	discover_innovation = innovation_plenary_assemblies 
 	discover_innovation = innovation_currency_01 
 	#
+	discover_innovation = innovation_taming_the_skies
+	discover_innovation = innovation_giants
+	discover_innovation = innovation_large_soldiers
```

Comment:
If you are going for ``large_soldiers`` on Eredar I think you should also do the same for Draenei.